<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FLSetWindow</class>
 <widget class="QMainWindow" name="FLSetWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>623</width>
    <height>246</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>623</width>
    <height>246</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>623</width>
    <height>246</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(198, 210, 218);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>60</y>
      <width>601</width>
      <height>141</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>第1组铁鞋</string>
     </attribute>
     <widget class="QRadioButton" name="radioButton">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>20</y>
        <width>89</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="autoExclusive">
       <bool>false</bool>
      </property>
     </widget>
     <widget class="QRadioButton" name="radioButton_2">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>50</y>
        <width>89</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="autoExclusive">
       <bool>false</bool>
      </property>
     </widget>
     <widget class="QLabel" name="label_6">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>80</y>
        <width>54</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>存车信息</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_10">
      <property name="geometry">
       <rect>
        <x>430</x>
        <y>20</y>
        <width>91</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string> 上行位置 警内</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_11">
      <property name="geometry">
       <rect>
        <x>430</x>
        <y>50</y>
        <width>91</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string> 上行位置 警内</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_14">
      <property name="geometry">
       <rect>
        <x>570</x>
        <y>20</y>
        <width>21</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>米</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_15">
      <property name="geometry">
       <rect>
        <x>570</x>
        <y>50</y>
        <width>21</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>米</string>
      </property>
     </widget>
     <widget class="QGroupBox" name="groupBox">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>45</y>
        <width>211</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">border:none</string>
      </property>
      <property name="title">
       <string/>
      </property>
      <property name="flat">
       <bool>false</bool>
      </property>
      <widget class="QRadioButton" name="radioButton_5">
       <property name="geometry">
        <rect>
         <x>90</x>
         <y>5</y>
         <width>121</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>人力制动机紧固器</string>
       </property>
      </widget>
      <widget class="QRadioButton" name="radioButton_3">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>5</y>
         <width>89</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>人力制动机</string>
       </property>
      </widget>
     </widget>
     <widget class="QGroupBox" name="groupBox_2">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>15</y>
        <width>211</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">border:none</string>
      </property>
      <property name="title">
       <string/>
      </property>
      <property name="flat">
       <bool>false</bool>
      </property>
      <widget class="QRadioButton" name="radioButton_6">
       <property name="geometry">
        <rect>
         <x>90</x>
         <y>5</y>
         <width>121</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>人力制动机紧固器</string>
       </property>
      </widget>
      <widget class="QRadioButton" name="radioButton_4">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>5</y>
         <width>89</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>人力制动机</string>
       </property>
      </widget>
     </widget>
     <widget class="QLineEdit" name="lineEdit">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>20</y>
        <width>54</width>
        <height>20</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(255, 255, 255);</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_2">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>50</y>
        <width>54</width>
        <height>20</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(255, 255, 255);</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_3">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>80</y>
        <width>54</width>
        <height>20</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(255, 255, 255);</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_4">
      <property name="geometry">
       <rect>
        <x>370</x>
        <y>20</y>
        <width>54</width>
        <height>20</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(255, 255, 255);</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_5">
      <property name="geometry">
       <rect>
        <x>370</x>
        <y>50</y>
        <width>54</width>
        <height>20</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(255, 255, 255);</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_6">
      <property name="geometry">
       <rect>
        <x>520</x>
        <y>20</y>
        <width>41</width>
        <height>20</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(255, 255, 255);</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_7">
      <property name="geometry">
       <rect>
        <x>520</x>
        <y>50</y>
        <width>41</width>
        <height>20</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(255, 255, 255);</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string>第2组铁鞋</string>
     </attribute>
     <widget class="QRadioButton" name="radioButton_7">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>20</y>
        <width>89</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QRadioButton" name="radioButton_8">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>50</y>
        <width>89</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="label_16">
      <property name="geometry">
       <rect>
        <x>570</x>
        <y>50</y>
        <width>21</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>米</string>
      </property>
     </widget>
     <widget class="QRadioButton" name="radioButton_9">
      <property name="geometry">
       <rect>
        <x>250</x>
        <y>50</y>
        <width>121</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>人力制动机紧固器</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_21">
      <property name="geometry">
       <rect>
        <x>430</x>
        <y>20</y>
        <width>91</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string> 上行位置 警内</string>
      </property>
     </widget>
     <widget class="QRadioButton" name="radioButton_10">
      <property name="geometry">
       <rect>
        <x>250</x>
        <y>20</y>
        <width>121</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>人力制动机紧固器</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_23">
      <property name="geometry">
       <rect>
        <x>570</x>
        <y>20</y>
        <width>21</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>米</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_25">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>80</y>
        <width>54</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>存车信息</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_27">
      <property name="geometry">
       <rect>
        <x>430</x>
        <y>50</y>
        <width>91</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string> 上行位置 警内</string>
      </property>
     </widget>
     <widget class="QRadioButton" name="radioButton_11">
      <property name="geometry">
       <rect>
        <x>160</x>
        <y>20</y>
        <width>89</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>人力制动机</string>
      </property>
     </widget>
     <widget class="QRadioButton" name="radioButton_12">
      <property name="geometry">
       <rect>
        <x>160</x>
        <y>50</y>
        <width>89</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>人力制动机</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_8">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>80</y>
        <width>54</width>
        <height>20</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(255, 255, 255);</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_9">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>50</y>
        <width>54</width>
        <height>20</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(255, 255, 255);</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_10">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>20</y>
        <width>54</width>
        <height>20</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(255, 255, 255);</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_11">
      <property name="geometry">
       <rect>
        <x>370</x>
        <y>50</y>
        <width>54</width>
        <height>20</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(255, 255, 255);</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_12">
      <property name="geometry">
       <rect>
        <x>370</x>
        <y>20</y>
        <width>54</width>
        <height>20</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(255, 255, 255);</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_13">
      <property name="geometry">
       <rect>
        <x>520</x>
        <y>20</y>
        <width>41</width>
        <height>20</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(255, 255, 255);</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_14">
      <property name="geometry">
       <rect>
        <x>520</x>
        <y>50</y>
        <width>41</width>
        <height>20</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(255, 255, 255);</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_3">
     <attribute name="title">
      <string>第3组铁鞋</string>
     </attribute>
     <widget class="QRadioButton" name="radioButton_13">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>20</y>
        <width>89</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QRadioButton" name="radioButton_14">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>50</y>
        <width>89</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="label_28">
      <property name="geometry">
       <rect>
        <x>570</x>
        <y>50</y>
        <width>21</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>米</string>
      </property>
     </widget>
     <widget class="QRadioButton" name="radioButton_15">
      <property name="geometry">
       <rect>
        <x>250</x>
        <y>50</y>
        <width>121</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>人力制动机紧固器</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_33">
      <property name="geometry">
       <rect>
        <x>430</x>
        <y>20</y>
        <width>91</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string> 上行位置 警内</string>
      </property>
     </widget>
     <widget class="QRadioButton" name="radioButton_16">
      <property name="geometry">
       <rect>
        <x>250</x>
        <y>20</y>
        <width>121</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>人力制动机紧固器</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_35">
      <property name="geometry">
       <rect>
        <x>570</x>
        <y>20</y>
        <width>21</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>米</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_37">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>80</y>
        <width>54</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>存车信息</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_39">
      <property name="geometry">
       <rect>
        <x>430</x>
        <y>50</y>
        <width>91</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string> 上行位置 警内</string>
      </property>
     </widget>
     <widget class="QRadioButton" name="radioButton_17">
      <property name="geometry">
       <rect>
        <x>160</x>
        <y>20</y>
        <width>89</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>人力制动机</string>
      </property>
     </widget>
     <widget class="QRadioButton" name="radioButton_18">
      <property name="geometry">
       <rect>
        <x>160</x>
        <y>50</y>
        <width>89</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>人力制动机</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_15">
      <property name="geometry">
       <rect>
        <x>370</x>
        <y>50</y>
        <width>54</width>
        <height>20</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(255, 255, 255);</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_16">
      <property name="geometry">
       <rect>
        <x>520</x>
        <y>50</y>
        <width>41</width>
        <height>20</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(255, 255, 255);</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_17">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>80</y>
        <width>54</width>
        <height>20</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(255, 255, 255);</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_18">
      <property name="geometry">
       <rect>
        <x>370</x>
        <y>20</y>
        <width>54</width>
        <height>20</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(255, 255, 255);</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_19">
      <property name="geometry">
       <rect>
        <x>520</x>
        <y>20</y>
        <width>41</width>
        <height>20</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(255, 255, 255);</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_20">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>50</y>
        <width>54</width>
        <height>20</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(255, 255, 255);</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_21">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>20</y>
        <width>54</width>
        <height>20</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(255, 255, 255);</string>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>54</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string> 股道</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>20</y>
      <width>261</width>
      <height>25</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>20</y>
      <width>41</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string> 操作员</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>20</y>
      <width>61</width>
      <height>25</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{background-color: rgb(240, 240, 240);}
QPushButton{border:1px groove gray;border-radius:4px;padding:2px 4px;border-style:outset;}
QPushButton:hover{background-color: rgb(229, 241, 251);color:black}
QPushButton:pressed{background-color: rgb(204, 228, 247);border-style:inset}
QPushButton{background-color: qlineargradient(spread:pad, x1:0, y1:1, x2:0, y2:0, stop:0 rgba(220, 220, 220, 255), stop:1 rgba(250, 250, 250, 255));}
QPushButton{text-align: left;}</string>
    </property>
    <property name="text">
     <string>无</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_2">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>210</y>
      <width>75</width>
      <height>25</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{background-color: rgb(240, 240, 240);}
QPushButton{border:1px groove gray;border-radius:4px;padding:2px 4px;border-style:outset;}
QPushButton:hover{background-color: rgb(229, 241, 251);color:black}
QPushButton:pressed{background-color: rgb(204, 228, 247);border-style:inset}
QPushButton{background-color: qlineargradient(spread:pad, x1:0, y1:1, x2:0, y2:0, stop:0 rgba(220, 220, 220, 255), stop:1 rgba(250, 250, 250, 255));}</string>
    </property>
    <property name="text">
     <string>移至</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_3">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>210</y>
      <width>75</width>
      <height>25</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{background-color: rgb(240, 240, 240);}
QPushButton{border:1px groove gray;border-radius:4px;padding:2px 4px;border-style:outset;}
QPushButton:hover{background-color: rgb(229, 241, 251);color:black}
QPushButton:pressed{background-color: rgb(204, 228, 247);border-style:inset}
QPushButton{background-color: qlineargradient(spread:pad, x1:0, y1:1, x2:0, y2:0, stop:0 rgba(220, 220, 220, 255), stop:1 rgba(250, 250, 250, 255));}</string>
    </property>
    <property name="text">
     <string>撤除</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_4">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>210</y>
      <width>75</width>
      <height>25</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{background-color: rgb(240, 240, 240);}
QPushButton{border:1px groove gray;border-radius:4px;padding:2px 4px;border-style:outset;}
QPushButton:hover{background-color: rgb(229, 241, 251);color:black}
QPushButton:pressed{background-color: rgb(204, 228, 247);border-style:inset}
QPushButton{background-color: qlineargradient(spread:pad, x1:0, y1:1, x2:0, y2:0, stop:0 rgba(220, 220, 220, 255), stop:1 rgba(250, 250, 250, 255));}</string>
    </property>
    <property name="text">
     <string>确定</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_5">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>210</y>
      <width>75</width>
      <height>25</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{background-color: rgb(240, 240, 240);}
QPushButton{border:1px groove gray;border-radius:4px;padding:2px 4px;border-style:outset;}
QPushButton:hover{background-color: rgb(229, 241, 251);color:black}
QPushButton:pressed{background-color: rgb(204, 228, 247);border-style:inset}
QPushButton{background-color: qlineargradient(spread:pad, x1:0, y1:1, x2:0, y2:0, stop:0 rgba(220, 220, 220, 255), stop:1 rgba(250, 250, 250, 255));}</string>
    </property>
    <property name="text">
     <string>取消</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
